src/OrchardCore.Modules/OrchardCore.Themes/Assets.json

```diff
@@ -7,8 +7,31 @@
   },
   {
     "inputs": [
-      "Assets/js/theme-manager.js"
+      "Assets/js/constants.js",
+      "Assets/js/theme-loader.js"
+    ],
+    "watch": [
+      "Assets/js/constants.js",
+      "Assets/js/theme-loader.js"
+    ],
+    "output": "wwwroot/Scripts/theme-head.js"
```

Comment:
`theme-head` because this should be loaded in the head to ensure we are loading theme early on. We also have theme manager but is it not utilized by OC themes.